app/models.py

```diff
@@ -819,12 +819,11 @@ class Notification(db.Model):
 
     @hybrid_property
     def status(self):
-        return self._status_enum
+        return self._status_fkey
```

Comment:
think we can just get rid of this property and rename `_status_fkey` to `status`